shelley/chain-and-ledger/formal-spec/incentives.tex

```diff
@@ -636,9 +648,29 @@ \subsection{How Rewards are returned to Owners and Delegators}
 \label{sec:rewards-return}
 
 In the Incentivised Testnet, rewards will be calculated following the end of the epoch,
-and returned to wallets as part of the transition to the MainNet.
-\khcomment{JC/MG to expand.}
-\khcomment{TODO: Verify that the Testnet rewards will not be added to stake pots. Still not clear from the Testnet documentation.}
+and returned to wallets as part of the transition to the MainNet.  This will be achieved
+using the instantaneous rewards return mechanism that is described below.
+
+\begin{itemize}
+  \item
+    When the testnet terminates, a snapshot will be taken in order to calculate the rewards that were earned.
+    The result will be a mapping of stake keys to rewards as a whole number of Lovelace.
+  \item
+    The MIR certificate (``move instantaneous rewards'') will contain a mapping of stake credentials (that are derived from the stake keys)
+    to Lovelace amounts.  These values will be auditable and visible on the blockchain.
+    Any  transaction containing one of these certificates must be entirely
+    signed by a \emph{quorum} of core nodes (currently set to be five or more).
+  \item
+    When the ledger processes a valid MIR certificate, it will ignore any stake credential that is not registered with the system.
+    All the stake keys in the snapshot will be registered by \IOHK{}. A deposit may be charged for the registration.
```

Comment:
should we be more explicit here? We could state that the deposit for the registration will be deduced from the available rewards that will be paid. 
With the above formulation it is not clear that a deposit has to be paid or who would pay it. Or is this deliberate?

Reply:
Will check with Jared.  We'd left it a bit vague because it wasn't clear what had actually been decided.

Reply:
It would be good to get sign-off on this, but I'm not sure who to ask. That being said, I think we will probably do what @mgudemann said above.